dateparser/calendars/jalali_parser.py

```diff
@@ -147,8 +147,8 @@ def comp_key(tup):
         day_pairs[-14] = thirty
         day_pairs[1] = thirteen
 
-        for persian, number in reduce(
+        for pers, number in reduce(
```

Comment:
:lipstick: This looks like a change for the worse to me. Any approach that does not require shortening the variable name?

Reply:
Yeah, that's right. What about  `from convertdate import persian as persian_calendar` (and the `persian, number` here) or leave it the import as-is and then change to `for persian_number, number in ...` ?

What approach do you like the most?

Reply:
I pushed a commit with a new version, let me know what you think :+1:

Reply:
Perfect, now looks much better!